aea/cli_gui/swagger.yaml

```diff
@@ -0,0 +1,215 @@
+swagger: "2.0"
+info:
+  description: This is the swagger file that goes with our server code
+  version: "1.0.0"
+  title: Swagger Rest Article
+consumes:
+  - application/json
+produces:
+  - application/json
+
+basePath: /api
+
+# Paths supported by the server application
+paths:
+  /agent:
+    get:
+      operationId: app.get_agents
+      tags:
+      - agents
+      summary: Return list of all aea projects
+      description: List of local folders under the user name (which correspond to aea projects)
+      responses:
+        200:
+          description: Successfully read agent list operation
+          schema:
+            type: array
+            items:
+              type: string
+    post:
+      operationId: app.create_agent
+      tags:
+        - People
+      summary: Create a person and add it to the people list
+      description: Create a new person in the people list
+      parameters:
+        - name: agent_id
+          in: body
+          description: Person to create
+          required: True
+          schema:
+            type: string
+
+      responses:
+        201:
+          description: Successfully created person in list
+
+        400:
+          description: Cannot create agent
+          schema:
+            type: string
+
+  /agent/{agent_id}:
+    delete:
+      operationId: app.delete_agent
+      tags:
+      - agents
+      summary: Delete an aea project
+      parameters:
+        - name: agent_id
+          in: path
+          description: id of agent to delete
+          type: string
+          required: True
+      responses:
+        201:
+          description: Agent deleted successfully
+          schema:
+            type: string
+
+        400:
+          description: Cannot delete agent
+          schema:
+            type: string
+
+  /agent/{agent_id}/{item_type}:
+    post:
+      operationId: app.fetch_item
```

Comment:
I think this should be called `add` rather than `fetch` so it stays consistent with the CLI naming conventions.